src/framework/uicomponents/qml/MuseScore/UiComponents/NavigationFocusBorder.qml

```diff
@@ -39,6 +39,6 @@ Rectangle {
     border.color: ui.theme.fontPrimaryColor
     border.width: ui.theme.navCtrlBorderWidth
     radius: Number(parent.radius) > 0
-            ? (drawOutsideParent ? parent.radius + root.border.width : parent.radius)
```

Comment:
Please explain why this change is needed?

Reply:
It fixes the corner radius of the navigation border for the thumbnails on the Scores page. In ScoreItem, we set the margin for the NavigationBorder explicitly, to create some padding between the border and the thumbnail, and with this change, the corner radius of the border is adjusted accordingly.

Reply:
What's wrong with the borders for the Score item in master? Looks nice

<img width="258" alt="image" src="https://user-images.githubusercontent.com/10116828/186859417-86081316-a530-446f-a75a-f7866d769362.png">

Reply:
It's subtle, _very_ subtle, but it is not entirely correct. I've tried to indicate the difference in this image:
<img width="592" alt="Scherm­afbeelding 2022-08-29 om 10 57 24" src="https://user-images.githubusercontent.com/48658420/187164210-c07f1dbc-b52e-49e1-9716-459a79e53740.png">
With the old version (left), it will look like the padding between the border and the thumbnail is a bit bigger in the corners. With the new version, it will look and feel like 2px all around.

Reply:
ok, thanks for clarification

Reply:
I decided to add a dedicated "padding" property to `NavigationFocusBorder`, to make it more clear what we're doing.